src/model/data/fridge/REST.yaml

```diff
@@ -19,8 +19,14 @@ tags:
   - name: tag
     description: Hash tag for filtering and searching
 
+servers:
+  - url: https://api-prod.communityfridgefinder.com/v1/
+    description: Production server (uses live data)
+  - url: https://api-dev.communityfridgefinder.com/v1/
+    description: Staging server (uses test data)
+
 paths:
-  /v1/fridges:
```

Comment:
What's the logic behind removing versioning?

Reply:
I have not removed versioning. The version is in the url path. I’ve added a server field so both the dev and prod AWS  lambdas can be queried directly from the Swagger Editor

Reply:
that's really cool :)